server/views/partials/layout.njk

```diff
@@ -92,13 +92,14 @@
 {% block main %}
   <div class="govuk-width-container {{ containerClasses }}">
     {{ govukPhaseBanner({
-      attributes: {
-        role: "complementary"
-      },
       tag: {
-        text: "beta"
+        text: "Beta"
       },
-      classes: "govuk-!-display-none-print"
+      html: 'This is a new service – get help or <a class="govuk-link" href="https://eu.surveymonkey.com/r/H9CYM7B" target="_blank" referrerpolicy="no-referrer" rel="noopener">give your feedback</a> to help us to improve it.',
```

Comment:
We don't want the URL hardcoded and embedded in the nunjucks view like this (I can almost guarantee they'll change it at some point! 🙄 )
The feedback URL needs to be an environment variable in the helm charts (it can be hardcoded there for the time being 👍 ), which in turn sets a property in the `config` object, which in turn is made available to the view layer.

Then the `html` property above becomes:
```
html: 'This is a new service – get help or <a class="govuk-link" href="{{ feebackUrl }}" target="_blank" referrerpolicy="no-referrer" rel="noopener">give your feedback</a> to help us to improve it.',
```